src/main/java/com/password4j/SaltGenerator.java

```diff
@@ -80,4 +80,23 @@ public static byte[] generate()
         return generate(64);
     }
 
+    /**
+     * Salt by definition are shared between all passwords and
+     * must be stored in a location different from the one used
+     * for the passwords.
+     * <p>
+     * It can be set in the <i>psw4j.properties</i> file with
+     * the property {@code global.salt.length}.
+     * <p>
+     * If the <i>psw4j.properties</i> or the property {@code global.salt.length}
+     * are not found, {@code null} is returned.
+     *
+     * @return a shared pepper set in the <i>psw4j.properties</i> file.
+     * @since 0.1.1
+     */
+    public static String get()
```

Comment:
change the signature to int.
Also this method must be called by [this method](https://github.com/Password4j/password4j/blob/c898bac51e712ae92ce9f8b5ec3c75f88370832e/src/main/java/com/password4j/SaltGenerator.java#L78) (remove the 64 and use this method)